Source/Lib/ASM_AVX2/EbComputeSAD_SadLoopKernel_AVX512.c

```diff
@@ -2180,7 +2180,7 @@ void GetEightHorizontalSearchPointResults_32x32_64x64_PU_AVX2_INTRIN(
 		bestSad64x64 = temSum;
 		bestMV64x64 = 7 * 4;
 	}
-	if (pBestSad64x64[0] != bestSad64x64) {
+	if (bestSad64x64 <= pBestSad64x64[0]) {
```

Comment:
And the one in Source/Lib/ASM_AVX2/EbComputeSAD_SadLoopKernel_AVX512.c although it's disabled...